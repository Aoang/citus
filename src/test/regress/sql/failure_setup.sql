SELECT citus.mitmproxy('conn.allow()');

-- add the workers
SELECT citus_add_node('localhost', :worker_1_port);
SELECT citus_add_node('localhost', :worker_2_proxy_port);  -- an mitmproxy which forwards to the second worker
