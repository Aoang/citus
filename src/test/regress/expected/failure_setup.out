SELECT citus.mitmproxy('conn.allow()');
 mitmproxy
---------------------------------------------------------------------

(1 row)

-- add the workers
SELECT citus_add_node('localhost', :worker_1_port);
 citus_add_node
---------------------------------------------------------------------
               1
(1 row)

SELECT citus_add_node('localhost', :worker_2_proxy_port);  -- an mitmproxy which forwards to the second worker
 citus_add_node
---------------------------------------------------------------------
               2
(1 row)

